src/smt/proof_manager.cpp

```diff
@@ -40,7 +40,7 @@ PfManager::PfManager(Env& env)
     : EnvObj(env),
       d_pchecker(new ProofChecker(
           options().proof.proofCheck == options::ProofCheckMode::EAGER,
-          options().proof.proofPedantic)),
+          static_cast<uint32_t>(options().proof.proofPedantic))),
```

Comment:
Shouldn't the option be `uint32_t` instead of having to use a cast from `uint64_t` -> `uint32_t`?

Reply:
I don't think we support different integer types at the options level (there was a compile error when I tried). Is this correct @nafur ?

Reply:
Yeah, we only support the 64bit variants.